nexusadspy/client.py

```diff
@@ -125,7 +125,6 @@ def _do_throttled_request(self, url, method, params=None, data=None, headers=Non
                               sec_sleep=2., max_failures=100,
                               get_field=None):
 
-        data = json.dumps(data)
```

Comment:
Because data won't convert it to JSON by default as required for some POST/PUT requests like report downloads or campaign creation.
removing this line may cause those calls to fail.